openslides/users/views.py

```diff
@@ -232,6 +238,22 @@ def get_context_data(self, **context):
             **context)
 
 
+class SetPasswordView(APIView):
+    """
+    Users can set a new password for themselves.
+    """
+    http_method_names = ['post']
+
+    def post(self, request, *args, **kwargs):
+        if request.user.check_password(request.data['old_password']):
+            user = request.user
+            user.set_password(request.data['new_password'])
+            user.save()
```

Comment:
Proposal:

Either move line 249 before line 248 and use user in 248 as well, or remove this line and work with request.user in line 250 and 251.